source/docs/articles/sites/backups/restoring-an-environment-from-a-backup.md

```diff
@@ -36,12 +36,23 @@ If you want to download a backup using wget, put the provided temporary link in
 
     wget "https://pantheon-backups.s3.amazonaws.com..."
 
+### Restoring The Codebase
```

Comment:
This is a subheading for 
`## Restore an Environment From Another Environment's Backup`, which isn't possible with clicking.